src/webapp/reports/autogenerated-forms/GridForm.tsx

```diff
@@ -117,27 +117,28 @@ const GridForm: React.FC<GridFormProps> = props => {
                             )}
                         </DataTableRow>
                     ))}
-                    {grid.summary && (
-                        <DataTableRow key="total-custom-row">
-                            <CustomDataTableCell
-                                backgroundColor={props.section.styles.totals.backgroundColor}
-                                key="total-column-name"
-                            >
-                                {grid.summary.cellName}
-                            </CustomDataTableCell>
-                            {grid.summary.cells.map(itemTotal => {
-                                return (
-                                    <DataTableCellFormula
-                                        key={itemTotal.columnName}
-                                        dataFormInfo={dataFormInfo}
-                                        styles={props.section.styles}
-                                        total={itemTotal}
-                                        formula={itemTotal.formula}
-                                    />
-                                );
-                            })}
-                        </DataTableRow>
-                    )}
+                    {grid.summary.length !== 0 &&
```

Comment:
`grid.summary` is an array so validation is not necessary anymore, right? An empty array will not render anything. Check the other files where summary is present as well please.